oak_functions_sdk/tests/integration_test.rs

```diff
@@ -160,6 +160,28 @@ async fn test_storage_get_item_not_found() {
     oak_functions_test_utils::assert_response_body(response, "No item found");
 }
 
+#[tokio::test]
+#[ignore]
+async fn test_storage_get_item_huge_key() {
+    let bytes: Vec<u8> = (0..(i32::pow(2, 20))).map(|_| 42u8).collect();
```

Comment:
a few nits (and similar below):

- There seems to be an unneeded bracket pair
- Should this not be a usize rather than i32?
- I think `vec![42u8; (1 << 20)]` or `vec![42u8; usize::pow(2, 20)]` is clearer